src/portfolio/APIClient.ts

```diff
@@ -2,6 +2,7 @@ import { AppError } from '../Errors';
 
 // TODO: Change to production URL when available
 // TODO: Use a common base URL for shell, analyser and portfolio instead of 3 URLs
+
 export const baseURL = 'https://api.milou.io/portfolio';
```

Comment:
These TODOs are for `baseURL`, so there is no need to add a blank line between.

```suggestion
// TODO: Change to production URL when available
// TODO: Use a common base URL for shell, analyser and portfolio instead of 3 URLs
export const baseURL = 'https://api.milou.io/portfolio';
```